x-pack/plugins/enterprise_search/public/applications/app_search/components/app_search_gate/app_search_gate.tsx

```diff
@@ -0,0 +1,620 @@
+      defaultMessage:
+        'Did you know the new self-managed Elastic open crawler is now available? You can keep your web content in sync with your search-optimized indices!',
+    }),
+    title: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.webCrawler.featureName', {
+      defaultMessage: 'Web Crawler',
+    }),
+  },
+  analyticsAndLogs: {
+    actionLabel: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.analyticsAndLogs.featureButtonLabel',
+      {
+        defaultMessage: 'Add Search Analytics',
+      }
+    ),
+    actionLink: 'https://www.elastic.co/guide/en/elasticsearch/reference/current/behavioral-analytics-event.html',
+    addOnLearnMoreLabel: undefined,
+    addOnLearnMoreUrl: undefined,
+    description: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.analyticsAndLogs.featureDescription',
+      {
+        defaultMessage:
+          "Add and view analytics and logs for your search application",
+      }
+    ),
+    id: 'analyticsAndLogs',
+    learnMore: 'https://www.elastic.co/guide/en/elasticsearch/reference/current/behavioral-analytics-overview.html',
+    panelText: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.analyticsAndLogs.panelText', {
+      defaultMessage:
+        'You can track and analyze users\' searching and clicking behavior with Behavioral Analytics. Instrument your website or application to track relevant user actions.',
+    }),
+    title: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.analyticsAndLogs.featureName', {
+      defaultMessage: 'Search Analytics and Logs',
+    }),
+  },
+  synonyms: {
+    actionLabel: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.synonyms.featureButtonLabel',
+      {
+        defaultMessage: 'Search with Synonyms',
+      }
+    ),
+    actionLink: 'https://www.elastic.co/guide/en/elasticsearch/reference/current/synonyms-apis.html',
+    addOnLearnMoreLabel: undefined,
+    addOnLearnMoreUrl: undefined,
+    description: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.synonyms.featureDescription',
+      {
+        defaultMessage:
+          "Perform search with synonym based query expansion",
+      }
+    ),
+    id: 'synonyms',
+    learnMore: 'https://www.elastic.co/guide/en/elasticsearch/reference/current/search-with-synonyms.html',
+    panelText: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.synonyms.panelText', {
+      defaultMessage:
+        'Use the Elasticsearch Synonyms APIs to easily create and manage synonym sets.',
+    }),
+    title: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.synonyms.featureName', {
+      defaultMessage: 'Search with Synonyms',
+    }),
+  },
+  relevanceTuning: {
+    actionLabel: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.relevanceTuning.featureButtonLabel',
+      {
+        defaultMessage: 'Tune Search Relevancy',
```

Comment:
Similar casing question, do we want `Tune search relevancy`? Or `Tune search relevance` might be more commonly used terminology plus more consistent with other references in the form